### DISCUSSION TITLE
Nextra's i18n keeps showing me 404 pages no matter what

### DISCUSSION THREAD

**Discussion by tobySolutions** [shuding/nextra#2646] [Q&A]
Sorry for barging in here like this, but, I have an issue with Nextra's i18n. I'm trying to enable support for i18n on the Million docs (https://million.dev/), but, I keep getting this weird 404 page thing even when I have followed exactly what is required:

https://github.com/aidenybai/million/pull/885

Thank you!

I have done a search for similar discussions and tried the fixes from the ones that had fixes, but, to no avail.

**Comment by dimaMachina** [comment]
I'll suggest try new i18n from v3 alpha https://github.com/shuding/nextra/issues/1421#issuecomment-1883980439

**Comment by tobySolutions** [comment]
> I'll suggest try new i18n from v3 alpha [#1421 (comment)](https://github.com/shuding/nextra/issues/1421#issuecomment-1883980439)

Thank you very much @dimaMachina!
